lib/engine/game/g_1880/step/dividend.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/dividend'
+require_relative '../../../step/minor_half_pay'
+
+module Engine
+  module Game
+    module G1880
+      module Step
+        class Dividend < Engine::Step::Dividend
```

Comment:
This class is fine; however, there is already a `minor_half_pay` module. A `minor_withhold` module would keep to that spirit and allow reuse. Up to you if you want to go that route.

FYI the `minor_half_pay` module has more code. Take a look and see if you need any of those other methods.

Reply:
added minor_withold